inc/alert.class.php

```diff
@@ -216,6 +216,69 @@ public function rawSearchOptions()
         return $tab;
     }
 
+    // @codingStandardsIgnoreStart
+    public function post_updateItem($history = true)
+    {
+        // @codingStandardsIgnoreEnd
+        if (
+            !isset($this->input['is_close_allowed'])
+            || $this->input['is_close_allowed'] !== '0'
+        ) {
+            return;
+        }
+
+        $userId = Session::getLoginUserID();
+        $alertUserId = $this->getAlertUserId($this->input['id'], $userId);
+
+        if ($alertUserId !== null) {
+            $this->updateAlertUserState($alertUserId, 0);
```

Comment:
shouldn't all the Alert_Users be updated?
Rather than just those of the logged-in user?

Reply:
Sure ! Fixed in [42afaf1](https://github.com/pluginsGLPI/news/pull/143/commits/42afaf178e2c7fdab7b7d8947b6df176c061d0c5) :+1: